gaj23/face-it
# Kby refactor components

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 
PWA: Added basic code skeleton to implement PWA and run a lighthouse audit.
Refactor/Fix Bugs: Add a Route and link to the search bar component.


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.
Added a few boilerplate files to implement the PWA technology. Also added our logo to the manifest.
Added a Route and link to the search bar form to render the display on a different page.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
App Component - added in a redirect for the Search bar component - only when you search will it take you to the search bar page
Search Results Component - added in a parenthesis to make sure the SearchResults as rendering. Also added in a Redirect for the search results and a Route. Created functionality to hide the search results once a user clicks on the other pages. Also set the state back once a user is done searching.

### How should this be tested?
> What steps need to be taken to confirm functionality? 
npm start => go to homepage and type in a search result and go home. Click around on the different nav bar elements and it should take you to the respective pages.

### Steps To Take
> Any questions and possible steps moving forward.



#### Checklist:
